rust/agama-server/src/l10n/web.rs

```diff
@@ -161,6 +161,7 @@ async fn set_config(
         state.manager_proxy.set_locale(&locale_string).await?;
         changes.ui_locale = Some(locale_string);
 
+        // TODO: reset/reread the software product cache, it contains translations
```

Comment:
The code works fine, the only problem is that the product cache needs to be read again after changing the language. Currently after changing the language the product descriptions are still displayed in English.